course/src/Vector3.cc

```diff
@@ -0,0 +1,156 @@
+#include "Vector3.h"
+
+namespace ekumen
+{
+
+Vector3::Vector3(const double x, const double y, const double z)
+{
+    values[0] = x;
+    values[1] = y;
+    values[2] = z;
+}
+Vector3::Vector3()
```

Comment:
Replace this constructor with a delegating constructor that uses the one above: https://www.learncpp.com/cpp-tutorial/8-6-overlapping-and-delegating-constructors/